junit-jupiter-api/src/main/java/org/junit/jupiter/api/condition/DisabledIf.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.api.condition;
+
+import static org.apiguardian.api.API.Status.STABLE;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apiguardian.api.API;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * {@code @DisabledIf} is used to signal that the annotated test class or
+ * test method is <em>disabled</em> only if the provided condition
+ * evaluates to {@code true}.
+ *
+ * <p>When applied at the class level, all test methods within that class
+ * will be disabled on the same specified operating systems.
```

Comment:
I believe you might have copied that from somewhere. 😉

Reply:
Oopsie, you got me! 😁